docker-compose.yaml

```
@@ -71,6 +71,7 @@ services:
         environment:
             - INFINITY_MODEL_ID=${INFINITY_EMBEDDING_MODEL};${INFINITY_RERANKING_MODEL}
             - INFINITY_BATCH_SIZE=8
+            - API_KEY=${INFINITY_API_KEY}
```

This seems incorrect?

```suggestion
            - INFINITY_API_KEY=${INFINITY_API_KEY}
```

---

Nope, Infinity service has API_KEY as env variable.